packages/react-emotion/typescript_tests/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
For glamorous we've added declaration: true to help make sure we're exporting everything required for use in libraries Microsoft/TypeScript#5938.

Reply:
Done! Added.
No warnings
:smile: